apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -413,6 +424,12 @@ class PageTemplateModal extends Component {
 					) : (
 						<>
 							<form className="page-template-modal__form">
+								{ this.props.isFrontPage &&
+									this.renderTemplatesList(
+										currentTemplate,
+										__( 'Current', 'full-site-editing' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **100** times:
`translate( 'Current' )` *ES Score: 10*